install-and-configure/advanced-configuration/gpu.md

```diff
@@ -308,3 +308,34 @@ kubectl -n kubecost port-forward svc/kubecost-prometheus-server 8080:80
 Open the Prometheus web interface in your browser by navigating to `http://localhost:8080`. In the search box, begin typing the prefix for a metric, for example `DCGM_FI_DEV_POWER_USAGE`. Click Execute to view the returned query and verify that there is data present. An example is shown below.
 
 ![Prometheus query showing DCGM Exporter metric](/images/gpu-prometheus-query.png)
+
+## Shared GPU Support
+
+Kubecost supports NVIDIA GPU sharing using either the CUDA [time-slicing](https://docs.nvidia.com/datacenter/cloud-native/gpu-operator/latest/gpu-sharing.html) or [Multi-Process Service (MPS)](https://docs.nvidia.com/deploy/mps/index.html) methods. MIG is currently unsupported but is being evaluated for a future release. When employing either time-slicing or MPS, you must use the `renameByDefault=true` option in the [NVIDIA device plugin's](https://github.com/NVIDIA/k8s-device-plugin?tab=readme-ov-file#shared-access-to-gpus) configuration stanza. This parameter instructs the device plugin to advertise the resource `nvidia.com/gpu.shared` on nodes where GPU sharing is enabled. Without this configuration option, the device plugin will instead advertise `nvidia.com/gpu` which will mean Kubecost is unable to disambiguate an "exclusive" GPU access request from a shared GPU access request. As a result, Kubecost's cost information will be inaccurate.
+
+The following is an example of a time-slicing configuration which sets the `renameByDefault` parameter.
```

Comment:
```suggestion
The following is an example of a time-slicing configuration which sets the `renameByDefault` parameter in the NVIDIA device plugin helm chart.
```

Not sure if the wording update is good, but I think it could be helpful to clarify what the below YAML block is configuring.

Reply:
Oh ok got it! Should we specify that somewhere? It's not immediately clear to me how to use the YAML from this paragraph alone. I had to dig through the `k8s-device-plugin` repo to better understand what this was for.

Reply:
I asked myself the same question but opted not to turn this into a full "let's teach you everything about NVIDIA GPU sharing soup to nuts" section because 1) it's a complicated subject and 2) it would be very lengthy. I think I'm ok, at least for now, leaving it with just the essentials and seeing where we go from here.

Reply:
Works for me!